### DISCUSSION TITLE
Onboard Renovate to target repositories without renovate.json

### DISCUSSION THREAD

**Discussion by larasika** [renovatebot/renovate#12153] [Request Help] [answered]
Hello !

So, in Gitlab I have 1 repository called `renovate-runner` for configuring Renovate and launching it to target repositories, and 2 target repositories where Renovate will actually run and update their dependencies.

I have set the default configuration (schedules, timezone, etc) in a `default.json` file in my `renovate-runner` repository and want to use it as a preset configuration in the target repositories, so each of them won't have individual `renovate.json` configured inside of themselves (the target repos) – is this possible ?
Or do we always need to have `renovate.json` in each individual repository for Renovate to be running?

I've tried to use `default.json` as a reference in the `.gitlab-ci.yml` file of my `renovate-runner` repo by adding 
```
RENOVATE_EXTENDS: 'gitlab>my-profile/renovate-runner'
```
Is this correct ?

tl;dr I'm trying to follow the steps here https://docs.renovatebot.com/config-presets/#gitlab-hosted-presets

I hope the question makes sense and someone could help me with this. Thank you !

**Comment by rarkins** [comment, marked as answer]
You should trying setting requireConfig=false and onboarding=false. Together it should mean that it runs on each repo regardless of whether there's a renovate.json.

**Reply by larasika** [reply]
This works ! Thank you very much.
Somehow I missed `requireConfig` in my configuration file 😅
